run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
```

Comment:
BTW My definition os a system container is just a container launched from a systemd unit file using runc.  But maybe other people here that name and thing of something different.

And to go with our other discussion it should probably be RunSystemdPod.

If we move to the Pod Method then we will need a way to join a Container to an existing POD.